e2e/playwright/support/utils.ts

```diff
@@ -14,13 +13,14 @@ export const waitForNewMessages = async (page: Page) => {
     // await page.locator('#postListContent').getByTestId('NotificationSeparator').getByText('New Messages').waitFor();
 };
 
-export const getTodoBotDMPageURL = async (client: Client4, teamName: string, userId: string) => {
-    let team = teamName;
-    if (team === '') {
-        const teams = await client.getTeamsForUser(userId);
-        team = teams[0].name;
-    }
-    return `${team}/messages/@todo`;
+export const getTeamName = async (client: Client4, userId: string) => {
+    const teams = await client.getTeamsForUser(userId);
+    const team = teams[0].name;
+    return team;
+};
+
+export const getBotDMPageURL = async (teamName: string, botUsername: string) => {
+    return `${teamName}/messages/${botUsername}`;
```

Comment:
I think we need the `@` symbol here right? Interesting how things are passing without this

```suggestion
    return `${teamName}/messages/@${botUsername}`;
```